ion/symboltable.go

```diff
@@ -202,7 +214,7 @@ func (s *sst) String() string {
 	buf := strings.Builder{}
 
 	w := NewTextWriter(&buf)
-	s.WriteTo(w)
+	_ = s.WriteTo(w)
```

Comment:
I think ```WriteTo()``` returns ```error```.
Why not checking the error whether it's nil or not?

Reply:
If this were inside a function that returns an error then I would have added the error checking. But it doesn't and I'm not gonna add an error to the return type because that would block sst from satisfying the `Stringer` interface.
I added this explicit `_ =` so that Goland would no longer flag this as a warning.